packages/test/test-end-to-end-tests/src/test/SummarizeFromLatest.spec.ts

```diff
@@ -0,0 +1,224 @@
+/*!
```

Comment:
Why is a separate file needed for this single test? Can't it be part of the above file?

Reply:
Removed the Latest as it does not add much value anymore.